src/contexts/auth/AuthContext.tsx

```
@@ -10,7 +10,12 @@ import {
   removeUserInfo,
   getKakaoUserInfo,
 } from "../../libs/kakaoAuth";
-import { loginWithOAuth } from "../../libs/api";
+import { 
+  loginWithOAuth, 
+  checkAuthStatus, 
```

ESLint

[error] 'checkAuthStatus' is defined but never used.

(@typescript-eslint/no-unused-vars)